internal/resume/templates/plaintext.go

```diff
@@ -1,47 +1,59 @@
 package templates
 
-import "text/template"
+import (
+  "text/template"
+  "strings"
+	"regexp"
+)
 
 // Plaintext returns the go template of the plaintext resume template
 func Plaintext() *template.Template {
-	tmpl, err := template.New("plaintext").Parse(plaintext)
+  fns := template.FuncMap{"censor": plaintextCensor}
+	tmpl, err := template.New("plaintext").Funcs(fns).Parse(plaintext)
 	if err != nil {
 		panic(err)
 	}
 
 	return tmpl
 }
 
+func plaintextCensor(s string) string {
+  re := regexp.MustCompile(`\|\|.*?\|\|`)
+  //func test(_s string) { return strings.Repeat("#",len(_s)-4) })
```

Comment:
Remove this commented out code. And don't forget to run `gofmt`!

Reply:
Woops.

I thought I `gofmt -w path/to/code.go`. Maybe I should automate it so that gofmt gets run before committing....

Reply:
`gofmt -s -w .` from the root directory ought to take care of it.